airflow/models/trigger.py

```diff
@@ -52,7 +52,7 @@ class Trigger(Base):
     id = Column(Integer, primary_key=True)
     classpath = Column(String(1000), nullable=False)
     kwargs = Column(ExtendedJSON, nullable=False)
-    created_date = Column(UtcDateTime, nullable=False)
+    created_date = Column(DateTime, nullable=False)  # DateTime without timezone
```

Comment:
I feel this _should_ use UtcDateTime.

Reply:
So this is a timezone-less datetime, in UTC? Maybe we can add that to the comment.

Reply:
It should end up as a timestamptz in the DB. How we get about there (in this PR or another) I don't mind

Reply:
Actually no, thinking a bit more about it: the model type was right, it was the DB that was wrong, so we should fix the DB in this PR for this case.